connector_search_engine/models/se_index.py

```diff
@@ -216,12 +216,14 @@ def _jobify_batch_sync(self, force_export: bool = False) -> None:
             description=description, identity_key=identity_exact
         ).batch_sync(force_export)
 
-    def _jobify_batch_recompute(self, force_export: bool = False) -> None:
+    def _jobify_batch_recompute(
+        self, force_export: bool = False, binding_ids: list | None = None
+    ) -> None:
         self.ensure_one()
         description = _("Prepare a batch recompute of index '%s'") % self.name
         self.with_delay(
             description=description, identity_key=identity_exact
-        ).batch_recompute(force_export)
+        ).batch_recompute(force_export, binding_ids=binding_ids)
```

Comment:
Strictly speaking this job description is now inaccurate because it is now a partial recompute? So maybe this method should be split too?